package.json

```diff
@@ -98,6 +98,7 @@
     ]
   },
   "dependencies": {
+    "prop-types": "^15.5.8",
     "react": "^15.0.0"
```

Comment:
Hey @kenny-hibino  yes i think changing the React version is a good idea. I'd be happy to commit that using `commitizen`. I am unfamiliar with the scope aspect of the commit, is there a preferred scope you would like me to choose?

thanks